XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -1141,7 +1147,7 @@
       <Version>6.1.9</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.4.0-prerelease.200203002</Version>
+      <Version>2.4.0-pr.200128002</Version>
```

Comment:
Does this update the nuget package to a lower version? Specifically to 20-01-28 (from 20-02-03)? Is this intentional?

Reply:
Just pushed an update with the latest pre-release nuget package. I was using the old one as it had the patch that Ranjesh pushed but it should be wrapped into the newest updated package by now (also tested it with the new package and it works).